src/Annotations/InjectUser.php

```diff
@@ -26,13 +28,13 @@ class InjectUser implements ParameterAnnotationInterface
     /**
      * @param array<string, mixed> $values
      */
-    public function __construct(array $values = [])
+    public function __construct(array $values = [], ?string $for = null)
```

Comment:
You can revert the changes in the constructor. The "for" parameter is only useful with Doctrine annotations. With PHP8 annotations, we will put the annotation on the parameter itself, which is way cleaner.

See https://graphqlite.thecodingmachine.io/docs/argument-resolving#annotations-parsing

Reply:
Makes sense! Pushed the adjustments.